### PR TITLE
Admin app: pin generation UX clarifications

### PR DESCRIPTION
**Pull request by jon-shipley**
Draft

### CODE REVIEW

File: admin/views/school/school-home.ejs [hunk 1]
```diff
@@ -34,7 +34,8 @@
                     <% } %>
                 </li>
             </ol>
-            <h3 class="govuk-heading-m govuk-!-margin-top-9">Preview the check</h3>
+            <!-- TODO JMS Delete this block -->
```
Comment (GuyHarwood) [inline admin/views/school/school-home.ejs:37]: superfluous code

**Review by GuyHarwood — CHANGES_REQUESTED** [review]

File: admin/views/school/school-home.ejs (hunk 1 above)
Comment (jon-shipley) [reply admin/views/school/school-home.ejs:37]: Yeah - it is at the moment.  It's a draft.

**Review by GuyHarwood — APPROVED** [review]
LGTM.  made one change which was due to vague ACs on my part (comma shifted after the pupils name on welcome screen)

**Review by GuyHarwood — APPROVED** [review]
LGTM

**Pull request merged**